src/rdbms-connect/azext_rdbms_connect/azext_metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.67"
```

Comment:
consider upgrading to latest azure cli version

Reply:
Updating to 2.15.1.